web/controllers/api/key_controller.ex

```diff
@@ -28,7 +28,7 @@ defmodule HexWeb.API.KeyController do
     multi =
       Ecto.Multi.new
       |> Ecto.Multi.insert(:key, Key.build(user, params))
-      |> Ecto.Multi.insert(:log, fn %{key: key} -> audit(conn, "key.generate", key) end)
+      |> audit_generate(user)
```

Comment:
@ericmj I just thought audits usage in controllers could be simplified to be just e.g.:

```elixir
defmodule ControllerHelpers do
  def audit(conn_or_user, action, params)
  def audit(multi, conn_or_user, action, fun)
end

Ecto.Multi.new
|> Ecto.Multi.insert(:key, Key.build(user, params))
|> audit(user, "key.generate", fn %{key: key} -> key end)

Ecto.Multi.new
|> Ecto.Multi.delete(:key, key)
|> audit(user, "key.remove", key)
```

this would remove boilerplate from the controllers, especially Multi.merge which is a bit noisy. WDYT?